openedx/features/course_experience/tests/views/test_course_home.py

```diff
@@ -89,7 +89,7 @@ def test_queries(self):
         course_home_url(self.course)
 
         # Fetch the view and verify the query counts
-        with self.assertNumQueries(45):
+        with self.assertNumQueries(46):
```

Comment:
LEARNER-531 also increments this from 45 to 47, so depending on the order in which we land the two branches - one will have to change this value.